### PR TITLE
[WIP] Fix: Minor alignment issue in Topology page

### PR DESCRIPTION
**Pull request by gijohn**
fixes https://jira.coreos.com/browse/ODC-1822

image

### CODE REVIEW

**Comment by spadgett** [comment]
It looks like this introduces a few other alignment issues. I'd like to remove any override for Patternfly style we have here, but I believe some other style changes will be needed.

cc @sg00dwin 

Import YAML · OKD 2019-10-10 09-29-50

Dashboards · OKD 2019-10-10 09-30-25

**Comment by spadgett** [comment]
+Add · OKD 2019-10-10 09-32-58

**Comment by gijohn** [comment]
> It looks like this introduces a few other alignment issues. I'd like to remove any override for Patternfly style we have here, but I believe some other style changes will be needed.
> 
> cc @sg00dwin
> 
> Import YAML · OKD 2019-10-10 09-29-50
> 
> Dashboards · OKD 2019-10-10 09-30-25

@spadgett, Looks like there are lots of page specific styles are applied. I'll continue to work on the styles.

**Comment by sg00dwin** [comment]
> > It looks like this introduces a few other alignment issues. I'd like to remove any override for Patternfly style we have here, but I believe some other style changes will be needed.
> > cc @sg00dwin
> > Import YAML · OKD 2019-10-10 09-29-50
> > Dashboards · OKD 2019-10-10 09-30-25
> 
> @spadgett, Looks like there are lots of page specific styles are applied. I'll continue to work on the styles.


I believe the following file changes should be made to get consistency across these views...


**_vars.scss**
`$grid-gutter-width: 27px; // match var(--pf-global--spacer--lg)`

**_edit-yaml.scss**
```
.yaml-editor__header {
  ...
  @media(min-width: $screen-sm-min) {
    padding-left: $grid-gutter-width;
  }
}
```

**_dropdown.scss**
```
 .co-namespace-bar {
  ...
  @media (min-width: $grid-float-breakpoint) {
    padding-left: $grid-gutter-width;
    padding-right: $grid-gutter-width;
  }
}
```

**dashboard.scss**
```
.co-dashboard-body {
  ...
  padding: $grid-gutter-width;
}
```

**Comment by gijohn** [comment]
@spadgett @sg00dwin I have made the necessary changes.

**Comment by spadgett** [comment]
It looks to me like mobile alignment is still off on at least some pages:

Dashboards · OKD 2019-10-11 10-48-46

**Comment by spadgett** [comment]
Do we want the heading and projects dropdown to align with the hamburger menu?

Pods · OKD 2019-10-11 10-51-08

**Comment by gijohn** [comment]
@spadgett, the menu alignment has 16px padding and it is defined in Patternfly. For mobile view, the content padding is default to 12px. How we can address it?

**Comment by spadgett** [comment]
Yeah, I see similar issues on the Patternfly site. We might raise this as a bug upstream. I don't know why you wouldn't want these to all align.

React | PatternFly 4 2019-10-11 11-39-47

**Comment by serenamarie125** [comment]
> Do we want the heading and projects dropdown to align with the hamburger menu?

@spadgett I think this is a good question for PatternFly - I'd assume we would want them to align

**Comment by spadgett** [comment]
I opened Patternfly issue https://github.com/patternfly/patternfly/issues/1190

File: frontend/public/style/_vars.scss [hunk 1]
```diff
@@ -27,8 +27,8 @@ $font-size-h3: var(--pf-global--FontSize--lg);
 $font-size-h4: var(--pf-global--FontSize--md);
 $font-size-h5: var(--pf-global--FontSize--md);
 $font-size-h6: var(--pf-global--FontSize--md);
-// Patternfly 3 defaults to 40px, which leaves a lot more whitespace than we're used to.
-$grid-gutter-width: 30px;
+// Patternfly 4 defaults to 24px
+$grid-gutter-width: 24px; // match var(--pf-global--spacer--lg)
```
Comment (rhamilto) [inline frontend/public/style/_vars.scss]: This change gives me pause.  We use `$grid-gutter-width` in a lot of places.

Comment (gijohn) [reply frontend/public/style/_vars.scss]: $grid-gutter-width is mostly used for padding and margin.  I believe this change will fix the alignment.

Comment (rhamilto) [reply frontend/public/style/_vars.scss]: @gijohn, I recommend against this change.  Changing `$grid-gutter-width` will have widespread implications across the console, and we shouldn't be making changes of this nature after feature freeze.  cc:  @spadgett

**Pull request closed without merging**